framework/doc/content/source/interfaces/Coupleable.md

```diff
@@ -1,31 +1,57 @@
 # Coupleable
 
-This class provides API for coupling different kinds of variables values into MOOSE systems.
-The following table summarizes the methods and kinds of values they provide:
-
-| Method | Description |
-| - | - |
-coupledValue | Values of a coupled variable in q-points
-coupledGradient | Gradients of a coupled variable in q-points
-coupledSecond | Second derivatives of a coupled variable in q-points
-adCoupledValue | Values of a coupled variable in q-points with automatic differentiation info
-adCoupledGradient | Gradients of a coupled variable in q-points with automatic differentiation info
-adCoupledSecond | Second derivatives of a coupled variable in q-points with automatic differentiation info
-coupledNodalValue | Values of a coupled variable at nodes.
-adCoupledNodalValue | Values of a coupled (vector) variable at nodes with automatic differentiation info
-coupledVectorValue | Values of a coupled vector variable in q-points
-adCoupledVectorValue | Values of a coupled vector variable in q-points with automatic differentiation info
-coupledCurl | Curl of a coupled vector variable in q-points
-coupledDot | Time derivative of a coupled variable
-coupledDotDu | Derivative of a time derivative of a coupled variable
-coupledNodalDot | Nodal value of the time derivative of a coupled variable
-coupledVectorDot | Time derivative of a coupled vector variable
-coupledVectorDotDu | Derivative of a time derivative of a coupled vector variable
-adCoupledLowerValue | Value a coupled lower-dimensional variable with automatic differentiation info
-
-For values, gradients and second derivatives, users can request old and older values in case they are running a transient simulation.
-In case of old and older values, the methods are called `coupledValueOld` and `coupledValueOlder`, respectively.
-Similarly,
+This class provides an API for coupling different kinds of variables values into MOOSE systems.
+The following tables summarize the methods it provides.
+
+| Methods for scalar variables | Description |
```

Comment:
scalar variables are their own thing, with a single dof for the entire domain
Maybe "scalar field variables" or "standard field variables" instead

Reply:
Done.